components/fields/radio.tsx

```diff
@@ -0,0 +1,41 @@
+import { FormikProps } from 'formik';
+import { ChangeEvent } from 'react';
+
+type IRadio = FormikProps<any> & {
+  id: string;
+  label: string;
+  // eslint-disable-next-line no-unused-vars
+  onBlur: (event: ChangeEvent<any>) => void;
```

Comment:
try to use the right event to avoid `any` wherever possible.

Reply:
ok sure. but here I am using formik that's why I keep it any to use the function for every field